src/components/Table.jsx

```diff
@@ -0,0 +1,606 @@
+                Title
+                </span>
+                <span>
+                <button onClick={handleTitleSort}>
+                  <img alt="ArrowSort icon" src={ArrowSort}/>                    
+                </button>
+                </span>
+            </>
+        );
+    }
+
+    const handleBrandSortHeader=()=>{
+        return (
+            <>
+                <span className="mr-2">
+                    Brand
+                </span>
+                <span>
+                <button onClick={handleBrandSort}>
+                 <img alt="ArrowSort icon" src={ArrowSort}/>                    
+                </button>
+                </span>
+            </>
+        );
+    }
+
+    const handleStatusSortHeader=()=>{
+        return (
+            <>
+                <span className="mr-2">
+                    Status
+                </span>
+                <span>
+                <button onClick={handleStatusSort}>
+                    <img alt="ArrowSort icon" src={ArrowSort}/>                    
+                </button>
+                </span>
+            </>
+        );
+    }
+
+    const handleAssigneeSortHeader=()=>{
+        return (
+            <>
+                <span className="mr-2">
+                    Assignee
+                </span>
+                <span>
+                <button onClick={handleAssigneeSort}>
+                    <img alt="ArrowSort icon" src={ArrowSort}/>                    
+                </button>
+                </span>
+            </>
+        );
+    }
+
+    const handleUpdatedBySortHeader=()=>{
+        return (
+            <>
+                <span className="mr-2">
+                    Updated By
+                </span>
+                <span>
+                <button onClick={handleUpdatedBySort}>
+                    <img alt="ArrowSort icon" src={ArrowSort}/>                    
+                </button>
+                </span>
+            </>
+        );
+    }
+
+    const handleUpdatedAtSortHeader=()=>{
+        return (
+            <>
+                <span className="mr-2">
+                    Updated At
+                </span>
+                <span>
+                <button onClick={handleUpdatedAtSort}>
+                   <img alt="ArrowSort icon" src={ArrowSort}/>                    
+                </button>
+                </span>
+            </>
+        );
+    }
+const handleFilterIcon=()=>{
+    return(
+        <span>
+            <button onClick={() => setShowFilters(!showFilters)}>
+                 {!showFilters ? <div className="bg-black text-white text-sm rounded-full border h-8 w-8 flex justify-center items-center"> 
+                   <i className="pi pi-filter" style={{fontSize: '0.8rem'}}/> 
+                 </div> :
+                 <div className="bg-white text-black text-sm rounded-full border h-8 w-8 flex justify-center items-center"> 
+                   <i className="pi pi-filter" style={{fontSize: '0.8rem'}}/> 
```

Comment:
consider using tailwind classes inside of such inline styles.
Same in other places also.

Reply:
addressed